modules/platforms/dotnet/Apache.Ignite.Core/Client/IClientClusterGroup.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Client
+{
+    using System;
+
+    /// <summary>
+    /// Defines grid projection which represents a common functionality over a group of nodes.
+    /// Grid projection allows to group Ignite nodes into various subgroups to perform distributed
+    /// operations on them. All ForXXX(...)' methods will create a child grid projection
+    /// from existing projection. If you create a new projection from current one, then the resulting
+    /// projection will include a subset of nodes from current projection.
+    /// </summary>
+    public interface IClientClusterGroup : IDisposable
```

Comment:
This interface seems to be not useful, it brings no value to the users. 

At lease we should have `GetNodes` here, which gives us some cluster inspection functionality. I propose to do this as a separate ticket and remove this interface for now.

Reply:
Removed for now